### ISSUE TITLE
Improve dependency TBB handling in CMake

### ISSUE THREAD

**Issue by redeboer** [ComPWA/ComPWA#270]
The current `CMakeLists.txt` file for the ThirdParty libraries does not perfectly handle finding TBB, see [here](https://github.com/ComPWA/ComPWA/blob/2fdf155ffacc214a8bbc9bff8984849d4b4f5eaa/ThirdParty/CMakeLists.txt#L3). This is, as commented, because of the buggy export of the TBB version we use, but the consequence is that TBB has to be rebuild each time you do a clean build of ComPWA, even if you already have system-wide install of TBB2019.

May have to wait for improvements in https://github.com/wjakob/tbb or contact them to improve the CMake export there (see [this advice in particular](https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/#if-you-want-it-done-right-do-it-yourself).

**Comment by spflueger** [comment]
I see. I will take a look at this

**Comment by redeboer** [comment]
Related to this point:
Many external headers are included as
`#include "ThirdParty/..."`
e.g. [here](https://github.com/ComPWA/ComPWA/blob/30c93db881b8b5e72f927b583c1d1eedcf33a2e7/Physics/EvtGen/EvtGenIF.hpp#L10), which does not allow one to use system-wide installs.

**Comment by spflueger** [comment]
> Related to this point:
> Many external headers are included as
> `#include "ThirdParty/..."`
> e.g. [here](https://github.com/ComPWA/ComPWA/blob/30c93db881b8b5e72f927b583c1d1eedcf33a2e7/Physics/EvtGen/EvtGenIF.hpp#L10), which does not allow one to use system-wide installs.

True. I think all third party stuff which well managed in terms of having also a cmake integration should be handled this way. This is not the case for EvtGen (maybe in the meantime) or qft++ (this is even slightly modified by us)

**Comment by weidenka** [comment]
I opened already an issue on that https://github.com/wjakob/tbb/issues/53
